pumps/splunk.go

```diff
@@ -199,6 +205,27 @@ func (p *SplunkPump) WriteData(ctx context.Context, data []interface{}) error {
 					continue
 				}
 
+				// Check if the current analytics field is "tags" and see if some tags are explicitly excluded
+				if (field == "tags" && len(p.config.IgnoreTagPrefixList) > 0) {
+					filteredTags := mapping["tags"].([]string)
+
+					// Loop all explicitly ignored tags
+					for _, excludeTag := range p.config.IgnoreTagPrefixList {
```

Comment:
@mbelchev this could be in a separated method for `SplunkPump` and have its own unit test. What do you think?

something like 

```go
func (p *SplunkPump) FilterTags(filteredTags []string) {
	// Loop all explicitly ignored tags
	for _, excludeTag := range p.config.IgnoreTagPrefixList {
		// Loop the current analytics item tags
		for key, currentTag := range filteredTags {
			// If the current tag's value includes an ignored word, remove it from the list
			if strings.HasPrefix(currentTag, excludeTag) {
				copy(filteredTags[key:], filteredTags[key+1:])
				filteredTags[len(filteredTags) - 1] = ""
				filteredTags = filteredTags[:len(filteredTags)-1]
			}
		}
	}
}
```

Reply:
Good point, it's now moved in a separate function. Thank you